README.rst

```diff
@@ -35,6 +35,6 @@ cd ukrdict
 ```
 3. Install ukrdict package
 ```bash
-setup.py install
+pip install -e git+https://github.com/serhii73/ukrdict.git#egg=ukrdict
```

Comment:
In general, since you're going to upload real dists I'd advise you to target PyPI instead of GitHub.